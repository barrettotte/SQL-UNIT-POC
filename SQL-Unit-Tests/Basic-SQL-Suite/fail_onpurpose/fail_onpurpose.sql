-- This is a simple test for for failing on purpose --

SELECT * FROM [BARRETT_TEST].[dbo].[FAIL_ME];