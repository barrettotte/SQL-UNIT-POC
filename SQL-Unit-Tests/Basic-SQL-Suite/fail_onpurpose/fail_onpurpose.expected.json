{
    "success": "true",
    "columns": [
        {
            "name": "WASD",
            "type": "VARCHAR(50)"
        }
    ],
    "rows": [
        {
            "WASD": "QWERTY"
        }
    ]
}